pkgs/by-name/py/pysunspec2/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  python3,
+  fetchPypi,
+}:
+
+python3.pkgs.buildPythonPackage rec {
```

Comment:
This should go in python-packages not by-name, to ensure it is configured correctly for all Python versions.